dashboard/config/course_offerings/hello-world-food.json

```diff
@@ -10,7 +10,7 @@
   "header": "hello_world",
   "image": "https://images.code.org/5021fd8558b98aac19d438d4eed1430f-Hello World Happy Food Theme.png",
   "cs_topic": "art_and_design,games_and_animations,programming",
-  "school_subject": "english_language_arts",
+  "school_subject": "",
```

Comment:
Checked that this matches what's on prod: https://github.com/code-dot-org/code-dot-org/blob/49d8e16ff6f41544b11a47490525901e6f2b93ee/dashboard/config/course_offerings/hello-world-food.json ✅